### PR TITLE
externalIP implementation for cluster ip service

### PR DESCRIPTION
**Pull request by hanamantagoudvk**
ExternalIP programming in IPVS , ipset , iptable is being taken care with this implementation

### CODE REVIEW

File: backends/ipvs-as-sink/common.go [hunk 1]
```diff
@@ -30,16 +31,56 @@ import (
...
+	AddEndPoint    Operation = 2
+	DeleteEndPoint Operation = 3
+)
+
 type endPointInfo struct {
 	endPointIP      string
 	isLocalEndPoint bool
 }
 
+func asDummyIPs(ip string, ipFamily v1.IPFamily) string {
```
Comment (qabpea) [inline backends/ipvs-as-sink/common.go:60]: Two things, 
First why the last ip + "/32" which other value can ipFamily have? Any value other than v4 or v6 should be an error.
Secondly, why the need to prepend a /32 or /128 scope at all? Do you just want to get "vip" address that you can hang on a loopback or use in route.
Given that the name of the function, this is just for my curiosity. No need really to change anything

**Review by qabpea — APPROVED** [review]
Looks good, I assume this is to sort out this one

[Fail] [sig-network] Services [It] should be able to change the type from ExternalName to ClusterIP [Conformance]
/workspace/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/network/service.go:1333

**Comment by jayunit100** [comment]
filed #194 as a follow on

**Pull request merged**